app/client/components/user/CreateUserForm.tsx

```diff
@@ -46,32 +51,39 @@ class CreateUserForm extends React.Component<InjectedFormProps<ICreateUser, Oute
 
     if (!window.IS_USING_BITCOIN) {
       uniqueIdentifierInput = (
-        <Field name="publicSerialNumber" component={TextInput} type="text" label={'Phone Number or ID'}>
+        <InputField name="publicSerialNumber" label={'Phone Number or ID'} isRequired>
           <QrReadingModal
             updateData={ (data: string) =>  this.setSerialNumber(data) }
           />
-        </Field>
+        </InputField>
       )
     } else {
-      uniqueIdentifierInput = <Field name="blockchainAddress" component={TextInput} type="text" label='Bitcoin Address' />
+      uniqueIdentifierInput = <InputField name="blockchainAddress" label='Bitcoin Address' isRequired />
     }
 
-
+    //TODO(admin_create): set a default for it to be easier for staff? should it be 400 or 40000 if we're giving 400 CICs?
     if (window.MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT > 0) {
-      initialDisbursementAmount = <Field name="customInitialDisbursement" component={TextInput} type="text" label={'Initial Disbursement Amount'}>
+      initialDisbursementAmount = <InputField name="customInitialDisbursement" label={'Initial Disbursement Amount'}>
         {window.CURRENCY_NAME}
-      </Field>
+      </InputField>
     }
 
+    //TODO(admin_create): add business category dropdown
+    //TODO(admin_create): add default token
+    //TODO(admin_create): add button to send SMS about short codes
     return (
       <div>
         <ModuleHeader>Create a {this.props.transferAccountType} account</ModuleHeader>
 
         <div style={{padding: '1em'}}>
           <form onSubmit={this.props.handleSubmit}>
-            <Field name="firstName" component={TextInput} type="text" label={'First Name'} />
-            <Field name="lastName" component={TextInput} type="text" label={'Last Name'} />
             {uniqueIdentifierInput}
+            <InputField name="firstName" label='Given Name(s)' isRequired />
+            <InputField name="lastName" label='Family/Surname' />
+            <InputField name="bio" label='Directory Entry' />
+            <InputField name="location" label='Location' />
+            <SelectField name="gender" label='Gender' options={["Male", "Female", "Other"]} />
```

Comment:
@RubenGeo this is where we'd put the business categories in the admin user creation interface, could be done similarly to gender